products/phpbb.md

```diff
@@ -0,0 +1,26 @@
+---
+title: phpBB
+category: server-app
+permalink: /phpbb
+releasePolicyLink: https://www.phpbb.com/community/viewtopic.php?f=14&t=2302466
+activeSupportColumn: false
+releaseColumn: true
+releaseDateColumn: false
+releases:
```

Comment:
https://github.com/phpbb/phpbb/tags could be used to automatically update latest patch versions using the following configuration:

```
-   git: https://github.com/phpbb/phpbb.git
    regex: ^release-(?<major>\d+)\.(?<minor>\d+)\.(?<patch>\d+)$
```

It's not supported yet, but we could use https://packagist.org/packages/phpbb/phpbb too.

Reply:
packagist only has 3.0.2+, so git tags is better (for now).